src/state/CalculationLogics.ts

```diff
@@ -26,53 +26,69 @@ export function getLengthOfLastPeriod(cycles: Cycle[]) {
     return 0;
   }
 
-  return Number(cycles[0].periodLength);
+  return cycles[0].periodLength;
 }
 
-export function getOvulationStatus(cycles: Cycle[]) {
+export function getDayOfCycle(cycles: Cycle[]) {
   if (cycles.length === 0) {
-    return "";
+    return 0;
   }
 
-  const cycleLength = getAverageLengthOfCycle(cycles);
-  const dayOfCycle = getDayOfCycle(cycles);
+  const start = startOfDay(new Date(cycles[0].startDate));
+  const currentDate = startOfToday();
 
-  const lutealPhaseLength = 14;
-  const ovulationDay = cycleLength - lutealPhaseLength;
-  const diffDay = ovulationDay - dayOfCycle;
-  if (diffDay < -2) {
-    return i18n.t("finished");
-  }
-  if (diffDay < 0) {
-    return i18n.t("possible");
-  }
-  if (diffDay === 0) {
-    return i18n.t("today");
-  }
-  if (diffDay === 1) {
-    return i18n.t("tomorrow");
-  }
-  return `${i18n.t("in")} ${diffDay} ${i18n.t("Days", {
-    postProcess: "interval",
-    count: diffDay,
-  })}`;
+  return differenceInDays(currentDate, start) + 1;
 }
 
-export function getPregnancyChance(cycles: Cycle[]) {
-  if (cycles.length === 0) {
-    return "";
-  }
+// NOTE: Detailed description of ovulation calculation: https://github.com/IraSoro/peri/blob/master/info/CALCULATION.md#ovulation-day
+export function getOvulationStatus(cycles: Cycle[]) {
+  if (cycles.length === 0) return "";
```

Comment:
Please add `{` and `}` here

If we'll try to debug this function in the future then we should add `{` `}` every time ourselves, why don't add them right away?